documentation/airport-format.md

```diff
@@ -1,309 +1,469 @@
+- **ctr_ceiling** ― The ceiling/top of the airspace (in feet). When an `airspace` property is present, that value will take priority over this one.
+- **initial_alt** ― The altitude (in feet) at which all departing aircraft are expected to stop their climb after takeoff unless otherwise instructed.
+- **position** ― The geographical position of the airport. (in latitude, longitude, and elevation: _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_)
+- **rr_radius_nm** ― The distance between each range ring (in nautical miles) within the airspace.
+- **rr_center** ― The position at which the range rings are centered. (in latitude, longitude: _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_)
+- **has_terrain** ― Flag used to determine if the airport has a corresponding `.geoJSON` file in [assets/airports/terrain](../..assets/airports/terrain).
+- **wind** ― The true heading (angle) in degrees and speed in knots of the current wind at the airport:
+```javascript
+"wind": {
+    "angle": 220,
+    "speed": 6
+}
+```
+
+
+### airspace
+ ```javascript
+ "airspace": [
+     {
+         "floor": 0,
+         "ceiling": 190,
+         "airspace_class": "B",
+         "poly": [
+             ["N35d57m50.000", "W115d51m15.000"],
+             ["N35d34m30.000", "W115d29m00.000"]
+         ]
+     }
+ ],
+```
+Position definition of the airport airspace.  Multiple airspace areas may be defined and will all be included in the airspace. This allows for advanced airspace stratification.
+- **_At least one airspace definition is required for an airport_**
+- **_All object properties are always required_**
+
+- **floor** ― The altitude (in feet) at which the airspace begins.
+- **ceiling** ― The altitude (in [FL][#flight-level] feet) at which the airspace ends.
+- **airspace_class** ― The FAA class of the airspace. _see [this FAA document](faa-airspace) for more details_
+- **poly** ― The coordinates of the airspace. in latitude, longitude: _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_
+
+
+### fixes
+```javascript
+"fixes": {
+    "_NAPSE068": ["N36.11211", "W115.14661"],
+    "BAKRR": ["N36.07582112978773", "W114.95309917207562"],
+    "BCE":   ["N37.68918661436860", "W112.30389943797489"],
+    "BESSY": ["N36.10772192196994", "W115.28956463349111"],
+    "BETHL": ["N36.88434886833625", "W112.44043432584908"],
+    "BIKKR": ["N36.56666216331978", "W116.75003219453492"]
+},
+```
+Each navaid located within or around the airport airspace in latitude, longitude: _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_.  Real life fixes are defined thusly:
+```javascript
+"BAKRR": ["N36.07582112978773", "W114.95309917207562"]
+```
+Sometimes, there is a need for a specific point in space that is not a real life fix.  In cases such as this, a fix can be created thusly:
+```javascript
+"_NAPSE068": ["N36.11211", "W115.14661"],
```

Comment:
Basically _fake_ waypoints to finagle the sim into doing things that it doesn't have the features to do "properly" (eg DME arcs around things), and _real_ waypoints that are only used as numbered fixes for use with a procedure and never used verbally (see EDDM for example). When a underscore-prefixed fix is the current waypoint, the strip just shows `RNAV` currently.